tests/test_l10n_download_links.py

```diff
@@ -7,74 +6,47 @@
 import pytest
 import requests
 
+pytestmark = [pytest.mark.nondestructive, pytest.mark.skip_selenium]
 
-nondestructive = pytest.mark.nondestructive
-skip_selenium = pytest.mark.skip_selenium
 
+def pytest_generate_tests(metafunc):
+    base_url = metafunc.config.option.base_url
+    page = metafunc.cls.page  # get the target page from the test class
+    r = requests.get('{0}{1}'.format(base_url, page))
+    soup = BeautifulSoup(r.content, 'html.parser')
+    table = soup.find('table', class_='build-table')  # find the table of links
+    idlist = []
+    argvalues = []
+    for row in table.find('tbody').find_all('tr'):
+        locale = row['id']
+        idlist.append(locale)  # the test id should be the locale
+        url = row.find('a')['href']
+        argvalues.append([locale, url])  # pass locale and url to each test
```

Comment:
Uber-nit: According to the docs [1] this should be a tuple, not a list.

[1] https://pytest.org/latest/parametrize.html#pytest-generate-tests